phoenix-pherf/src/main/java/org/apache/phoenix/pherf/workload/MultiThreadedRunner.java

```diff
@@ -131,14 +142,16 @@ private void timedQuery() throws Exception {
         Long start = EnvironmentEdgeManager.currentTimeMillis();
         Date startDate = Calendar.getInstance().getTime();
         String exception = null;
-        long resultRowCount = 0;
+        Long resultRowCount = 0L;
+        String queryIteration = threadName + ":" + iterationNumber;
+        Boolean timedOut = false;
 
         try {
             conn = pUtil.getConnection(query.getTenantId(), scenario.getPhoenixProperties());
             conn.setAutoCommit(true);
             final String statementString = query.getDynamicStatement(ruleApplier, scenario);
             statement = conn.prepareStatement(statementString);
-            LOGGER.info("Executing: " + statementString);
+            LOGGER.info("Executing " + queryIteration + ": " + statementString);
```

Comment:
I added it because I feel like it's useful to know which execution of the query we're on so you know how many were completed before a potential failure, but you can also see that in the output files, so I can remove it if it's not necessary

Reply:
I think that's fine, maybe change the log line to "Executing iteration : " + queryIteration ...